examples/register_xpub/register_xpub.go

```diff
@@ -14,12 +15,22 @@ func main() {
 	// Create a client
 	buxClient, _ := buxclient.New(
 		buxclient.WithXPriv(keys.XPriv()),
-		buxclient.WithHTTP("localhost:3001"),
+		buxclient.WithHTTP("http://localhost:3003/v1"),
```

Comment:
Is the `http://` part necessary here? I see an inconsistency here, because in the other example the `url` is without it.

Reply:
I agree with Krzysiek, the rest is ok for me

Reply:
Removed